src/http/register-websocket/connection.js

```diff
@@ -48,14 +48,12 @@ module.exports = function connection (params, connectionId, ws) {
   ws.on('close', function close () {
     let lambda = get.ws('disconnect')
     update.status(`ws/disconnect: ${connectionId}`)
+    pool.delete(connectionId)
```

Comment:
Are we sure we want to delete the connection first? I think this breaks this: https://github.com/architect/sandbox/blob/main/src/http/invoke-ws/_req-fmt.js#L11

Reply:
Followup: you are probably aware but reverting this change breaks the `Internal Arc API Gateway Management services` tests. I'll go ahead and merge with a minor patch to this logic.